playground/src/error-list/error-list.js

```diff
@@ -61,73 +65,55 @@ class ErrorList {
     this.validationResult = validationResult;
     window.requestIdleCallback(() => {
       /* eslint-disable max-len */
-      this.container.innerHTML = `
-        <div class="title">
-        <button class="button close">
-          <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
-            <path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/>
-            <path d="M0 0h24v24H0z" fill="none"/>
-          </svg>
-        </button>
-        <h2>Validation Result</h2>
-        </div>
-        <ul>${validationResult.errors.map(this.renderError).join('')}</ul>
-        `;
+      const content = document.createElement('ul');
+      for (var i = 0; i < validationResult.errors.length; i++) {
+        const error = validationResult.errors[i]
+        content.appendChild(this.renderError(error, i))
+      }
+
       /* eslint-enable max-len */
       if (validationResult.errors.length === 0) {
-        this.container.classList.toggle('show', false);
+        this.hideFlyIn();
       }
+
+      this.upadateContent(content);
     });
   }
 
   renderError(error, index) {
-    return `<li class="validation-error ${error.icon}" data-index="${index}">
-            <div>
-              <div class="message">${error.message}</div>
-              <div class="category">${error.category}</div>
-              <div class="location">line ${error.line}, column ${error.col}</div>
-            </div>
-      </li>`;
-  }
+    const errorElement = document.createElement('li');
+    errorElement.className = `validation-error ${error.icon}`;
+    errorElement.setAttribute('data-index', `${index}`);
 
-  toggle(e) {
-    if (this.container.classList.contains('show')) {
-      this.hideErrorList(e);
-    } else {
-      this.showErrorList(e);
-    }
-  }
+    errorElement.innerHTML = (
+      `<div>
+        <p class="message">${error.message} <a href="${error.specUrl}" target="_blank" rel="noopener">Learn&nbsp;more</a></p>
+        <div class="location">line ${error.line}, column ${error.col}</div>
+      </div>`
+    );
 
-  hideErrorList() {
-    document.body.removeEventListener('click', this.onItemClickHandler, false);
-    this.container.classList.toggle('show', false);
-  }
+    errorElement.addEventListener('click', (e) => {
```

Comment:
Remove the anonymous function wrapper, simply use `this.onErrorItemClick.bind(this)`